decrypt/action.yml

```diff
@@ -10,14 +10,42 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: actions/setup-node@v3
-      with:
-        node-version: 16
-    - run: npm install -g --unsafe-perm @ironcorelabs/ironhide
+    - name: Set env vars (Linux)
+      if: runner.os == 'Linux'
+      shell: bash
+      run: |
+        echo FILE_PATTERN='*linux-musl*' >> $GITHUB_ENV
+        echo UNZIP_CMD='tar -xf *linux-musl*' >> $GITHUB_ENV
+    - name: Set env vars (macOS)
+      if: runner.os == 'macOS'
       shell: bash
-    - run: mkdir -p ~/.iron
+      run: |
+        echo FILE_PATTERN='*apple-darwin*' >> $GITHUB_ENV
+        echo UNZIP_CMD='tar -xf *apple-darwin*' >> $GITHUB_ENV
+    - name: Set env vars (Windows)
+      if: runner.os == 'Windows'
       shell: bash
-    - run: echo '${{ inputs.keys }}' > ~/.iron/keys
+      run: |
+        echo FILE_PATTERN='*windows-gnu*' >> $GITHUB_ENV
+        echo UNZIP_CMD='7z x *windows-gnu*' >> $GITHUB_ENV
```

Comment:
I know you asked before, but I'm looking at it now and we may want msvc here. It's 3.51 MB vs 8.67 MB for the GNU one. If all other things are equal (which we're guessing at), and given our internal use case for the action, it's probably worth using the smaller one.

Reply:
I pushed a change and tested it and seems to work fine :+1: